style.css

```diff
@@ -17,6 +17,10 @@ body {
   background-repeat: no-repeat;
 }
 
```

Comment:
- [ ] Am really sorry, this point should be mentioned in previous projects, The mobile version breakpoint is in the range of `main 375px max 786px`, with this not being covered before now the project is not behaving responsively, for that I highly suggest to options:
- Avoid setting fixed width to all of the sections, which is what the mobile-first approach is really about.
- Or add another media query for the viewport in `main 375px max 786px`

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221282600-823e6707-9d39-404e-bd81-ed95ba0d1968.png)